org.eclipse.lsp4xml.extensions.maven/src/main/java/org/eclipse/lsp4xml/extensions/maven/MavenCompletionParticipant.java

```diff
@@ -210,9 +210,9 @@ private void completeProperties(ICompletionRequest request, ICompletionResponse
 		allProps.put("project.groupId", project == null ? "unknown" : project.getGroupId());
 		allProps.put("project.artifactId", project == null ? "unknown" : project.getArtifactId());
 		allProps.put("project.name", project == null ? "unknown" : project.getName());
-		allProps.put("project.build.directory", project == null ? "unknown" : project.getBuild().getDirectory());
+		allProps.put("project.build.directory", project.getBuild() == null ? "unknown" : project.getBuild().getDirectory());
```

Comment:
There's already a null check that occurs for the project instance occuring beforehand, so we can assume project !=null